safespace/src/components/SafeSpace.js

```diff
@@ -67,37 +88,34 @@ class SafeSpace extends React.Component {
             }
         };
 
-        const body = {
-            body : message,
-            scheduled : new Date(),
-        }
-
-        console.log('body', body);
+        // const body = {
+        //     body : message,
+        //     scheduled : time,
+        // }
        
-        axios.post(url,body,headers)
+        axios.post(url,message,headers)
              .then( res => {
                  console.log('new message response: ', res.data);
-                //  const newMessage = {
-                //      ...res.data,
-                //      body : this.state.newMessage
-                //  }
                  this.setState({
                      messages : [...this.state.messages, res.data]
-                 })                   
+                 })
+                 this.sms({body:'message will be sent'},'+14694143109‬',phone);
              })
              .catch( err => console.log('new message error', err))
 
+        setTimeout(this.sms, duration, message,'+14694143109‬',phone); 
```

Comment:
i like the idea here as a work around, but does the setTimeout actually still operate when the user navigates away from the app? for instance, if I do a duration that would text me tomorrow then i log out of the app, will that text actually come through? just curious